lib/alerts_viewer_web/components/core_components.ex

```diff
@@ -636,10 +636,16 @@ defmodule AlertsViewerWeb.CoreComponents do
   end
 
   @doc """
-  Renders a date-time using human-friendly formatting.
+  Renders a date-time in Eastern time using human-friendly formatting.
   """
   attr(:date_time, :map, required: true)
 
+  def date_time(%{date_time: %DateTime{}} = assigns) do
+    ~H"""
+    <%= friendly_date_time(@date_time |> DateTime.shift_zone!("America/New_York")) %>
```

Comment:
```suggestion
    <%= @date_time |> DateTime.shift_zone!("America/New_York") |> friendly_date_time() %>
```
suggestion: I think structuring the pipes like this would make it more idiomatic and potentially more readable.

As an alternate approach, you could also move the `DateTime.shift_zone!("America/New_York")` bit into `friendly_date_time`, which would let you avoid pipes in this component.

Reply:
I'll move it in there and see what it looks like.

Reply:
Ok, moving it looks a lot better. Thanks!